en/docs/ipaas/micro-integrator/developing-integrations-with-integration-studio.md

```diff
@@ -0,0 +1,107 @@
+# Developing Integrations with Integration Studio
+
+Choreo iPaaS offers a platform for developing and deploying integrations with ease. With the Integration Studio,
+developers can create integrations using WSO2 Synapse and deploy them in a Micro Intgrator (MI) runtime. Choreo iPaaS
+provides a centralized management system for integrations, making deployment and maintenance simple.
+
+## Integration Types
+
+Choreo supports following types of MI Integrations
```

Comment:
It is better to give a brief description for each integration type, and to explain what "MI" means here.

Suggestion:
"Choreo supports following types of MI (Micro Integrator) integrations:

!!! info
    MI integration refer to an integration you develop using WSO2 Synapse.

- **Integration as an API**: These integrations exposed as APIs using HTTP.
- **Event-triggered Integration**: Integrations that are triggered by external events.
- **Scheduled Integration**: Integrations that are triggered automatically at scheduled time intervals
- **Manually-triggered Integration**:Integrations that you need to trigger manually.
- **Composite Integration (Coming soon)**"